### ISSUE TITLE
Fire Sound effect

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#129]
Make the fire "haha" sound effect not play when touching burners or podoboos (lava bubbles)

**Comment by TheMan6900** [comment]
You do know that this causes lag right?

**Comment by EmeraldLoc** [comment]
No it doesnt. That's probably a tmk thing.

**Comment by TheMan6900** [comment]
> No it doesnt. That's probably a tmk thing.

You know that my thing doesnt cause lag right?

**Comment by TheMan6900** [comment]
Oh yeah this can also cause softlocks

**Comment by TheMan6900** [comment]
s o f t l o c k s

**Comment by EmeraldLoc** [comment]
A sound effect does not cause soft locks im sorry, its something else, what do you mean?

**Comment by TheMan6900** [comment]
well its not the sound effect but the collision of the flame/any fire thing

**Comment by EmeraldLoc** [comment]
Huh? Is this even related to Tag?

**Comment by TheMan6900** [comment]
well yeah, the weird way tag handles flame collision causes unintended interactions.

**Comment by EmeraldLoc** [comment]
Huh? The only thing tag does is cancel an interaction with a flame, nothing else, how does this softlock you?

**Comment by TheMan6900** [comment]
In some states, yes.

**Comment by EmeraldLoc** [comment]
Can you send a video of this, I don't know what that means

**Comment by TheMan6900** [comment]
dont really know how to do it, just take my word that it does indeed causes the bad stuffs.

**Comment by EmeraldLoc** [comment]
Oh my, WHAT causes bad stuff? WHAT is this custom tag interaction you're talking about? I have no information to go off of, the descriptions are too vague. Any interaction with a flame is cancelled so I don't know what you're referring to. I believe you're talking about something completely different but I don't know what it is. I have no information for what to look at to reproduce the bugs here, other than its related to a silly flame.

**Comment by TheMan6900** [comment]
the bad stuff I was talking about was softlocks, thought you would get what I was saying, whoops.
also this is the interaction that I was talking about 


https://github.com/user-attachments/assets/99524b94-a96f-419d-80ff-ea8bf6304b6b

**Comment by EmeraldLoc** [comment]
> the bad stuff I was talking about was softlocks

Yes, I understand this, what causes the soft locks? I have no info for what causes that.

> also this is the interaction that I was talking about

So the flame not disappearing is the issue? Thats clear enough

**Comment by TheMan6900** [comment]
pretty sure how it happened is from picking up a chuckya, than getting burned.

**Comment by EmeraldLoc** [comment]
I don't recall a romhack that has that combination. You do romhack stuff, can you get a video of this happening?

**Comment by TheMan6900** [comment]
ok nevermind it doesnt do anything...

**Comment by TheMan6900** [comment]
https://github.com/EmeraldLoc/tag/issues/63#issue-2304636163

**Comment by EmeraldLoc** [comment]
I want to say this is fixed, I don't notice any issues with fire, so yea.

Edit: I misinterpreted this issue, I'm remembering what the problem was
